src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/architecture/ArtisticArchitectureForm.tsx

```diff
@@ -180,7 +180,7 @@ export const ArtisticArchitectureForm = () => {
             </>
           )}
         </FieldArray>
-        <Box sx={{ display: 'flex', gap: '1rem', mt: '0.5rem' }}>
+        <Box sx={{ display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, gap: '1rem', mt: '0.5rem' }}>
```

Comment:
Hva med wrap i stedet?

```suggestion
        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '1rem', mt: '0.5rem' }}>
```

![bilde](https://github.com/user-attachments/assets/1d3158f6-68a4-4a1d-86f9-270cc543ff16)
vs
![bilde](https://github.com/user-attachments/assets/cb3f40a0-2873-4c31-b3ff-9873c6025608)

Reply:
Tror jeg går for å beholde `flexDirection` her, da jeg synes det ser best ut "overall". Om det skulle komme flere elementer sentere, er det ikke en spesielt stor fiks å endre/legge til `flexWrap` på container (evt. `flexGrow`)